README.md

```diff
@@ -65,6 +65,14 @@ And optionally, to install into $GOPATH/bin:
 make install
 ```
 
+## Building checkpoint image
+
+From the project root, run:
+
+```
+docker build -t $(REPO):$(VERSION_TAG) -f cmd/checkpoint/Dockerfile .
```

Comment:
We possibly could just include this tool in the bootkube image also... it's already just a scratch image with static binary. And if so, could be added to build/build-release script.